src/Controller/UI/AccountController.php

```diff
@@ -0,0 +1,514 @@
+<?php
+
+namespace App\Controller\UI;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+use App\Entity\Account;
+use App\Entity\Entity;
+use App\Entity\Password;
+use App\Entity\Person;
+use App\Entity\PersonToken;
+use App\Event\EntityEventDispatcher;
+use App\Handler\EntityHandler;
+
+
+
+class AccountController extends AbstractController
+{
+    /**
+     * Protected entityHandler value instance of entityHandler.
+     *
+     * @var entityHandler
+     */
+    protected $entityHandler;
+
+    /**
+     * Protected validator value instance of Symfony Validator.
+     *
+     * @var validator
+     */
+    protected $validator;
+
+    /**
+     * Constructor for this Controller, to set up default services.
+     */
+    public function __construct(EntityHandler $entityHandler, ValidatorInterface $validator)
+    {
+        $this->entityHandler = $entityHandler;
+        $this->validator = $validator;
+    }
+
+    /**
+     * The index action.
+     *
+     * @Route("/account", methods={"GET"}, name="pelagos_app_ui_account_default")
```

Comment:
I don't think we need to continue using the old names for the UI routes

Reply:
In the future we should change them, i need them for now.